mixer/pkg/config/crd/store.go

```diff
@@ -111,13 +110,18 @@ type Store struct {
 var _ store.Backend = &Store{}
 var _ probe.SupportsProbe = &Store{}
 
+// Close implements io.Closer
```

Comment:
Do we have to implement io.Closer? I don't think we're using this interface to reference the Store.

It looks to me, if we simply have a Close() method that returns nothing, we can simplify some of the client code paths.